pages.it/common/cp.md

```diff
@@ -0,0 +1,27 @@
+# cp
+
+> Copia files e directory
```

Comment:
```suggestion
> Copia file e directory.
```

The terminal -s is not used for English plurals in Italian. I see you used this form extensively, so I'm only commenting this particular case to not repeat myself, but please **apply the correction to any other English plural in other pages too** (e.g. `bytes`).